internal/pkg/constants/constants.go

```diff
@@ -97,10 +97,6 @@ const (
 
 	// TalosConfigEnvVar is the environment variable for setting the Talos configuration file path.
 	TalosConfigEnvVar = "TALOSCONFIG"
-
-	// ContainerBasedResolvConf is a resolv.conf that can be used for container based deploys.
-	// See https://github.com/coredns/coredns/tree/master/plugin/loop#troubleshooting
-	ContainerBasedResolvConf = "nameserver 8.8.4.4\nnameserver 8.8.8.8"
```

Comment:
Do we need to gen this from dhcp or does it get auto injected via magic of docker compose?

Reply:
Docker compose magic. Actually we should not do any networking when in container mode. Another future improvement.